app.js

```diff
@@ -0,0 +1,12 @@
+require("dotenv").config();
+require("./database");
+
+const express = require("express");
+const app = express();
+const router = require("./routing");
```

Comment:
Il manque pas le /index ?
```suggestion
const router = require("./routing/index");
```